be/CMakeLists.txt

```diff
@@ -490,6 +492,9 @@ endif()
 if (${WITH_CACHELIB} STREQUAL "ON")
     set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -DWITH_CACHELIB")
 endif()
+if (${WITH_STARCACHE} STREQUAL "ON")
```

Comment:
if provided both `WITH_CACHELIB=ON` and `WITH_STARCACHE=ON`, does it means both cache libraries will be used? Do they need to be exclusive?